logIn2/app/src/main/java/com/example/lmontesanto/login/MainActivity.java

```diff
@@ -0,0 +1,13 @@
+package com.example.lmontesanto.login;
+
+import android.app.Activity;
+import android.os.Bundle;
+
+public class MainActivity extends Activity {
```

Comment:
We should replace the view for the fragment . look at the ExampleActivity and ExampleFargment